client/blocks/get-apps/apps-badge.jsx

```diff
@@ -97,7 +97,7 @@ export class AppsBadge extends PureComponent {
 
 	onLinkClick = () => {
 		const { storeName } = this.props;
-		recordTracksEvent( APP_STORE_BADGE_URLS[ storeName ].tracksEvent );
+		analytics.tracks.recordEvent( APP_STORE_BADGE_URLS[ storeName ].tracksEvent, {} );
```

Comment:
In the interest of continuing to use the analytics middleware, instead of switching to use `analytics` here you could continue to import `recordTracksEvent` but use this to dispatch the action:

```
dispatch( recordTracksEvent( APP_STORE_BADGE_URLS[ storeName ].tracksEvent ) );
```

This works to record the tracks event in my testing. WDYT?

Reply:
If that's the more correct way to do it, I vote we go that way for sure! I wasn't aware that it worked that way. I'll update the PR to work this way tomorrow!

Reply:
This should be the connected dispatch prop:
```suggestion
		this.props.recordTracksEvent( APP_STORE_BADGE_URLS[ storeName ].tracksEvent, {} );
```

That doesn't exist yet, see notes in my review.